src/api/app/views/webui/comment/_reply.html.haml

```diff
@@ -18,8 +18,9 @@
 - if User.session
   .collapse{ id: "reply_form_of_#{comment.id}" }
     = render(partial: 'webui/comment/comment_field', locals: { form_method: :post,
-     comment: comment.children.new, commentable: commentable })
+     comment: comment.children.new, commentable: commentable, element_suffix: "reply_for_#{comment.id}" })
   - if policy(comment).update?
     .collapse{ id: "edit_form_of_#{comment.id}" }
       = render(partial: 'webui/comment/comment_field', locals: { form_method: :put,
-       comment: comment, commentable: commentable })
+       comment: comment, commentable: commentable, element_suffix: "edit_#{comment.id}" })
+      %button.btn.cancel.btn-outline-gray-dark Cancel
```

Comment:
Is this expected, it doesn't look right:

![cancel-button-missing-style](https://user-images.githubusercontent.com/1102934/98964360-01333a80-2509-11eb-99ca-c1462dd0fedc.png)

Reply:
Oups! It will be in another PR and with correct style:) Sorry, removed the button. Thank you.